src/tests/base_test.py

```diff
@@ -829,3 +830,8 @@ def check_status():
             sleep_seconds=30,
             waiting_for="controller to be running",
         )
+
+    @staticmethod
+    def get_ocp_cluster_nodes(kubeconfig: str):
```

Comment:
This method shouldn't be part of this class/module.
From my perspective it can be part of the changes on https://gitlab.cee.redhat.com/ocp-edge-qe/kni-assisted-installer-auto/-/merge_requests/702

Reply:
Removed from this PR.